polaris-core/src/main/java/org/apache/polaris/core/config/PolarisConfigurationStore.java

```diff
@@ -118,9 +118,14 @@ public interface PolarisConfigurationStore {
       PolarisConfiguration<T> config) {
     if (config.hasCatalogConfig() || config.hasCatalogConfigUnsafe()) {
       Map<String, String> propertiesMap = catalogEntity.getPropertiesAsMap();
-      String propertyValue = propertiesMap.get(config.catalogConfig());
+      String propertyValue = null;
```

Comment:
nit: stylistically, these changes would be a great one-liners, but NBD.

Reply:
Good idea. We didn't have this luxurious ternary operator in Scala :)

Reply:
Wait, unresolving -- unfortunately the linter formatted the second one as:
```
        propertyValue =
            config.hasCatalogConfigUnsafe()
                ? propertiesMap.get(config.catalogConfigUnsafe())
                : null;
```

Is this still preferable to:
```
if (config.hasCatalogConfigUnsafe()) {
  propertyValue = propertiesMap.get(config.catalogConfigUnsafe());
}
```

Reply:
😢 maybe not then...it only made it even longer...